config/install/field.field.node.films.field_characters.yml

```diff
@@ -0,0 +1,27 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_characters
+    - node.type.films
+id: node.films.field_characters
+field_name: field_characters
+entity_type: node
+bundle: films
+label: Characters
+description: 'A list of characters that in this film.'
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:node'
+  handler_settings:
+    target_bundles:
+      title: title
```

Comment:
Looks like this field is referencing the content type we removed. This field needs to be updated to specify the "People" content type.
![film_characters_reference_type](https://github.com/notS0Solo/swapimigration/assets/1434979/609e73d4-7cb7-4936-a681-809eb59b7a9e)